app/views/procurements/new.html.erb

```diff
@@ -0,0 +1,30 @@
+<section>
+  <nav>
+    <div>
+      <%= link_to '<', category_procurements_path %>
+    </div>
+    <div class="add-page-text">
+      <h2>Add a New Transaction</h2>
+    </div>
+    <%= button_to 'Sign Out', destroy_user_session_path, method: :delete %>
+  </nav>
+  <section class="main">
+    <%= form_with model: @procurement, html: { multipart: true }, url: category_procurements_path do |f| %>
+      <div>
+        <%= f.text_field :name, placeholder: "Name of transaction" %>
+      </div>
+      <div>
+        <%= f.number_field :amount, placeholder: "Amount" %>
+      </div>
+      <div class="field">
+        <%= f.label :category_ids, 'Select Categories' %>:
+        <div>
+          <%= f.collection_check_boxes :category_ids, Category.all, :id, :name %>
+        </div>
+      </div>
```

Comment:
![image](https://github.com/Wineshuga/buddy_budget/assets/93039359/92414cbd-e60e-4e75-950f-d0f7a32601ed)


- [ ] Great work on making the `Add Transaction` form but there is an authorization problem 😨 Because when I try to add a transaction to a category I also see categories of other users in the options 🤔 Kindly fix this issue so we can only see our own categories 👍